src/Identity/test/InMemory.Test/FunctionalTest.cs

```diff
@@ -26,7 +26,7 @@ namespace Microsoft.AspNetCore.Identity.InMemory
 {
     public class FunctionalTest
     {
-        const string TestPassword = "1qaz!QAZ";
+        const string TestPassword = "[PLACEHOLDER]-1a";
```

Comment:
Is it worth considering having a shared constant somewhere so if we ever need to change this again there won't be as many diffs?